lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -19,6 +19,13 @@ class GlobalCollectGateway < Gateway
       self.money_format = :cents
       self.supported_cardtypes = %i[visa master american_express discover naranja cabal]
 
+      THREEDS_EXEMPTIONS = {
+        moto: 'MOTO',
+        mail: 'MAIL',
+        telephone: 'TELEPHONE',
+        ecommerce: 'ECOMMERCE'
```

Comment:
I'm surprised there's `moto`, `telephone`, and `mail` since MOTO stands for `mail order telephone order`. Does the gateway specify when to use `moto` vs the other two?

Reply:
I know but it doesn't specify when to use which. Now I'm actually thinking about removing `ecommerce` since it says [that's the default value](https://apireference.connect.worldline-solutions.com/s2sapi/v1/en_US/dotnet/payments/create.html?paymentPlatform=GLOBALCOLLECT).

Reply:
> MAIL - The transaction is a Mail Order.
MOTO - The transaction is a Mail Order/Telephone Order.
TELEPHONE - The transaction is a Telephone Order.

Did the merchant specifically reqest to add `MAIL` and `TELEPHONE`? If not then I'd suggest we just add MOTO

Reply:
We want to support all available exemption types which is why I added `mail` and `telephone` but it does seem redundant with them also supporting MOTO